crates/partition-store/src/partition_store_manager.rs

```diff
@@ -55,13 +62,20 @@ impl PartitionStoreManager {
     pub async fn create(
         mut storage_opts: impl LiveLoad<Live = StorageOptions> + 'static,
         initial_partition_set: &[(PartitionId, RangeInclusive<PartitionKey>)],
+        persisted_lsn_tx: Option<watch::Sender<(PartitionId, Lsn)>>,
```

Comment:
Why do I need to send the sender in? is it possible to always create the sender and allow me to subscribe directly from `PartitionStore`?

Reply:
Removed, thanks for the suggestion!